packages/blocks/src/api/factory.js

```diff
@@ -134,13 +100,15 @@ export function createBlocksFromInnerBlocksTemplate(
 export function cloneBlock( block, mergeAttributes = {}, newInnerBlocks ) {
 	const clientId = uuid();
 
+	const sanitizedAttributes = sanitizeBlockAttributes( block.name, {
```

Comment:
The addition of sanitize could potentially have a  small performance impact (not sure if noticeable), the other thing is that it was not an expectation of cloneBock and I hope it doesn't unexpected produce side effects

Reply:
Does running `npm run test-performance` with and without this PR demonstrate any performance regression?